libraries/AP_BattMonitor/AP_BattMonitor_DroneCAN.cpp

```diff
@@ -137,7 +137,9 @@ void AP_BattMonitor_DroneCAN::update_interim_state(const float voltage, const fl
 
     const uint32_t tnow = AP_HAL::micros();
 
-    if (!_has_battery_info_aux || _mppt.is_detected) {
+    if (!_has_battery_info_aux ||
+        _mppt.is_detected ||
+        option_is_set(AP_BattMonitor_Params::Options::Ignore_UAVCAN_SoC)) {
```

Comment:
We do have a `use_CAN_SoC` method, it adds a check for soc 127 but is other wise the same. 
```suggestion
        || !use_CAN_SoC()) {
```